packages/core/src/Utils/Analytics/index.ts

```diff
@@ -23,6 +23,23 @@ export const AnalyticsInitializer = async () => {
                           utm_content: 'no content',
                       }
                     : Cookies.getJSON('utm_data');
+
+            /*
+                Temp solution to fetch country and handle experiments without causing inconsistencies to the users. 
+            */
+
+            const getCountry = async () => {
```

Comment:
There's a risk that country can return as empty string. What will Growthbook do with that?

Reply:
it will not match and the feature will be turned off